File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/03. Fitness Card/03. Fitness Card.js
function fitnessCard(money, gender, age, sport){
 
const gim = {
        'm' :{'Gym' : 42, 'Boxing' : 41, 'Yoga' : 45, 'Zumba' : 34, 'Dances' : 51, 'Pilates' : 39},
        'f' :{'Gym' : 35, 'Boxing' : 37, 'Yoga' : 42, 'Zumba' : 31, 'Dances' : 53, 'Pilates' : 37}
      }

let subtotal = gim[gender][sport]
age <= 19 ? subtotal *= 0.80 : null
total = subtotal - money

if (money >= subtotal)
    return `You purchased a 1 month pass for ${sport}.`
return `You don\'t have enough money! You need $${total.toFixed(2)} more.`

}

//console.log(fitnessCard(50, 'm', 23, 'Gym'))

File: Python Exams/01. Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/03. Fitness Card/03. Fitness Card.py
money = float(input())
gender = input()
age = int(input())
sport = input()

gim = {
        'm' :{'Gym' : 42, 'Boxing' : 41, 'Yoga' : 45, 'Zumba' : 34, 'Dances' : 51, 'Pilates' : 39},
        'f' :{'Gym' : 35, 'Boxing' : 37, 'Yoga' : 42, 'Zumba' : 31, 'Dances' : 53, 'Pilates' : 37}
      }

subtotal = gim[gender][sport]

if age <= 19:
 subtotal = subtotal * 0.80

total = abs(money - subtotal)

if money >= subtotal:
    print(f'You purchased a 1 month pass for {sport}.')
else:
    print(f'You don\'t have enough money! You need ${total:.2f} more.')  